### ISSUE TITLE
log panel "sync with cursor" to make it easier to correlate graphs with logs

### ISSUE THREAD

**Issue by pokemane** [grafana/grafana#21357]
**What would you like to be added**:

I'd like for the Logs panel to use the "shared crosshair" dashboard option, to sync the selected/focused line with the graph crosshairs. It'd also want to autoscroll the log panel to have that line in the center of the panel vertically.

**Why is this needed**:

It'd really help with the usability of the log panel, to be able to more easily correlate log events with the related graphs elsewhere on the dashboard.

I know that there might be some weirdness when you're hovering over a line in the log, where it might not be exactly clear where to put the crosshair or tooltip on the graphs. I'd be okay with it being "one-way" so the log panel can't pop the crosshairs up on the graphs, but the graphs *can* highlight or focus a line in the log panel.

**Comment by aocenas** [comment]
@davkal this seems like an interesting feature and maybe not even that complicated. Do you know whether something like this was already discussed or requested somewhere else?

**Comment by davkal** [comment]
@aocenas yup, this absolutely sounds helpful. It's not discussed anywhere else AFAIK. I've been wanting to do something similar across Explore splits, but it's not written down anywhere.

**Comment by rplevka** [comment]
I was just about to open the same RFE (including autoscroll). I was surprised this wasn't implemented yet. It would be really helpful!
